Dockerfile

```diff
@@ -80,4 +111,4 @@ ENV UWSGI_SOCKET=127.0.0.1:6002
 USER $USER
 
 ENTRYPOINT ["/entrypoint"]
-CMD ["runserver"]
+CMD ["runserver", "127.0.0.1:8000"]
```

Comment:
Totally nitpicking, but `"127.0.0.1:8000"` is the default

@vegu I'm thinking we should assign ports to each ctl service so they run together without needing to config each port?

Reply:
Yup this line can come out

Reply:
Fixed in https://github.com/fullctl/ixctl/pull/38/commits/7cb9e0cd7285023ca68f5d0f722153587a816fb3

Reply:
@grizz yeah makes sense, lets discuss the ports in slack, resolving this since the nitpick was fixed